packages/core/src/indexing/sqlite/__tests__/sqlite-index-api.test.ts

```diff
@@ -111,8 +106,8 @@ describe('indexStream', () => {
     expect(raw.stream_id).toEqual(STREAM_ID_B)
     expect(raw.controller_did).toEqual(CONTROLLER)
     expect(raw.last_anchored_at).toBeNull()
-    const createdAt = new Date(raw.created_at * 1000)
```

Comment:
why did this change?

Reply:
SQLite does not _really_ handle timestamps. So, the decision how to store timestamps is up to a developer. Before this change we assumed second precision. That can lead to problems for a developer, if she does multiple insertions during a single second. A situation easy to imagine. Insertion ordering in this case would be effectively broken for her, as order is undefined for the entries having the same timestamp.

Just in case, PG does not suffer, as it has microseconds precision.